config/api/routes/lock.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * Content Lock Routes
+ */
+return [
+
+    [
+        'pattern' => '(:all)/lock',
+        'method'  => 'GET',
+        'action'  => function (string $path) {
+            return $this->parent($path)->lock(false, $this->language());
+        }
+    ],
+    [
+        'pattern' => '(:all)/lock',
+        'method'  => 'PATCH',
+        'action'  => function (string $path) {
+            return $this->parent($path)->lock($this->requestBody('user'), $this->language());
```

Comment:
I think it would be better to use the currently authenticated user here (as well as for the `unlock` routes below), not a user that gets passed from the client. After all it should always only be possible to lock an object to the current user.